python-sdk/tests/test_sdk.py

```diff
@@ -1,19 +1,32 @@
 from typing import List, Dict
 import pytest
-from rebuff.sdk import RebuffSdk
+from rebuff.sdk import RebuffSdk, VectorDB
 from .utils import get_environment_variable
 
 
 @pytest.fixture()
-def rebuff() -> RebuffSdk:
+def rebuff(request) -> RebuffSdk:
     rb = RebuffSdk(
         get_environment_variable("OPENAI_API_KEY"),
+        request.param,
         get_environment_variable("PINECONE_API_KEY"),
         get_environment_variable("PINECONE_INDEX_NAME"),
     )
```

Comment:
For most test methods, it doesn't do any good testing on both pinecone and chroma; only `test_detect_injection_vectorbase` needs to test with both. You could do this in the fixture:
```suggestion
def rebuff(request) -> RebuffSdk:
    vector_db = request.param if hasattr(request, "param") else VectorDB.PINECONE
    rb = RebuffSdk(
        get_environment_variable("OPENAI_API_KEY"),
        vector_db,
        get_environment_variable("PINECONE_API_KEY"),
        get_environment_variable("PINECONE_INDEX_NAME"),
    )
```
Which would allow you to delete:
```py
@pytest.mark.parametrize(
    "rebuff",
    [VectorDB.PINECONE, VectorDB.CHROMA],
    ids=["pinecone", "chroma"],
    indirect=True,
)
```
from most of the methods.